src/frontend/src/lib/components/hero/Header.svelte

```diff
@@ -21,6 +22,13 @@
 	{:else}
 		<div class="flex p-4 items-center text-off-white">
 			<OisyWalletLogo />
+			{#if authNotSignedIn}
+				<div
+					class="absolute max-w-[3rem] leading-none text-[8px] uppercase font-semibold translate-x-[112%] translate-y-[130%] hidden md:flex"
```

Comment:
- For consistency, can we use a standard font size, like text-sm, and not introduce en discrepancy?

- Shouldn't this section be an `::after` that way it does not push the content given that we display it only on wider screen?

Reply:
- the smallest tailwind size` text-xs` is 12px; I used it at first but I thought it was a bit too much

### 12px
<img width="1076" alt="Screenshot 2024-07-22 at 11 12 30" src="https://github.com/user-attachments/assets/5a5cc410-0ca2-44f5-94d8-8524b3307686">

### 8px
<img width="1080" alt="Screenshot 2024-07-22 at 11 12 57" src="https://github.com/user-attachments/assets/a299e685-272e-407b-bedd-b8f10be3b598">

WDYT?


- I am not really sure how to do that? you mean to create a new class

```css
.text-never-download::after {
    content: {$i18n.hero.text.never_download}
    position: absolute;
    ....
}
```

and then use it as

```svelte
<div class="flex p-4 items-center text-off-white"
class:text-never-download={$authNotSignedIn}
>
	<OisyWalletLogo />
</div>
```

Reply:
Thanks for trying. Let's keep it as such then.

Reply:
That said, didn't Artem agree to not display this slogan on mobile? 
Not sure what was the last final decision?

```
class="hidden md:block"
```

Reply:
indeed in the end it was decided to not showing it. corrected!